### ISSUE TITLE
elasticity graph 

### ISSUE THREAD

**Issue by Ellie7** [Ellie7/parrot-proj#19]
a few issues with this graph: 
- I can't manually set the scale on the y axis so that it starts at 0 ( I have tried to do this with line 212 of code - + ylim(0, max(elas))) 
- can't change legend title either (code lines 213-215) 
- should I add lines also when there are so few points 

couldn't work out how to link to code so here is a copy! 

```
### plotting figure 3 - plot the proportional sensitivity to changes in F, P and G 
stage <- c("E", "J", "A")
#F <- c(elasticity[1, 1:3]) if including F1 & F2 on graph 
F <- c(NA, NA, elasticity[1,3])
#P <- c(elasticity[1,1], elasticity[2,2], elasticity[3,3]) if including P1 
P <- c(NA, elasticity[2,2], elasticity[3,3])
G <- c(elasticity[2,1], elasticity[3,2], NA)
sensitivities <- data.frame(stage, F, P, G)
sensitivity <- read.csv("cheat for now.csv") 
sens <- gather(sensitivities, vr, elas, F, P, G)
mutate(sens, "vr" = "Vital rate")
#change sensitivites data frame into the correct format 
fig <- ggplot(sens, aes(x = stage, y = elas, colour = vr, vr)) + geom_point(size = 7) + labs(x = "Stage", y = "Elasticity", size = 20)
fig.3 <- fig + scale_x_discrete(limits=c("E","J","A"), labels=c("Egg", "Juvenile", "Adult"))
figur.3 <- fig.3 + theme(axis.title = element_text(size = 14))
figure.3 <- figur.3 + scale_fill_discrete(name = "Vital rate") +
  ylim(0, max(elas))
figure.3
figure.3 + scale_fill_discrete(name="Vital rate",
                              breaks=c("F", "G", "P"),
                              labels=c("Reproductive Output", "Survival with growth to the next stage", "Survival while remaining in the same stage"))
```

**Comment by tdjames1** [comment]
Re your issues:
* Your elasticity is stored as `elasticity` since you removed the rounding, so you need to modify `ylim(0, max(elas)))` accordingly.

* You need to use `scale_colour_discrete()` to adjust the colour of points (`fill` is for filled areas. `colour` is for points and lines).

* Andrew mentioned on Slack that you don't need to add lines now that you've removed the zero points. Just make sure you can explain it well :)

Couple of other suggestions (at your discretion)
* Your legend could have simpler labels, e.g. Reproduction, Survival + Growth, Survival
* When you build up the graph you can reuse the same name, e.g.
``` 
fig <- ggplot(...)
fig <- fig + ...
fig <- fig + ...
```

**Comment by Ellie7** [comment]
Thank you all really helpful! all sorted with that one now :) fixed with commit 453d87f

**Comment by tdjames1** [comment]
Great :)
